pkg/kube/inject/inject.go

```diff
@@ -139,6 +139,10 @@ const (
 	ProxyContainerName = "istio-proxy"
 )
 
+const (
+	MTLSReadyLabelName = "security.istio.io/mtlsReady"
```

Comment:
comments on exported var?

Reply:
@incfly was hoping for some feedback on the placement of this variable - related issue https://github.com/istio/istio/issues/16439
Is `pkg/kube/inject/` the right place for this var?  I know all the annotations were moved to a common place in istio/api repo - do we have any other labels we track in istio?

Reply:
can you move this to model? esp since it is consumed by service entry as well and it will be required by eds as well